resources/js/components/AttributeList.vue

```diff
@@ -5,10 +5,14 @@
         item-key="id"
         class="attribute-list-container align-content-start"
         :class="classes"
+        ghost-class="vue-draggable-ghost"
```

Comment:
I wanted to use multiple bootstrap classes, but it's not possible. See in _style.scss
which then uses bootstrap variables.